actors/builtin/miner/miner_actor.go

```diff
@@ -2937,9 +2937,9 @@ func resolveWorkerAddress(rt Runtime, raw addr.Address) addr.Address {
 	return resolved
 }
 
-func burnFunds(rt Runtime, amt abi.TokenAmount) {
+func burnFunds(rt Runtime, amt abi.TokenAmount, bt BurnType) {
 	if amt.GreaterThan(big.Zero()) {
-		rt.Log(rtt.DEBUG, "storage provder %s burning %s", rt.Receiver(), amt)
+		rt.Log(rtt.INFO, "storage provder %s burn type %s burning %s", rt.Receiver(), bt, amt)
```

Comment:
Must be debug as this will be very noisy

Reply:
```suggestion
		rt.Log(rtt.DEBUG, "storage provder %s burn type %s burning %s", rt.Receiver(), bt, amt)
```

Reply:
BurnFunds method logs only appear when burn occurs in Miner Actor. Generally, debug logs are not enabled in non-test environments, and burn information is a very important part of burnFunds method logs.

Reply:
Because reward 、transfer and withdraw are important,  and burn is also important, need to check the account.

Reply:
We wonder what type of method causes burn.

Reply:
No disagreement that it is important.  But it is not worth the tradeoff.  Burning is a common operation across all miner actors.  All lotus nodes will be overrun with log noise if we keep at info.  

The right thing to work on to make this better is an extension to actors logging that subscribes to logging at different levels per actor.

Reply:
The burn method is important because you can filter the burn content by grep command on the log file.

Reply:
I understand but I'm not going to merge this with log level info

Reply:
Don't get the burn method name in Lotus, only value.

Reply:
Right, and with the suggested change you will now get the burn method name if you turn on actors debug logs.

Reply:
debug mode  enabled will be very noisy. Other debug logs are not required.

Reply:
The server running Lotus enables info-level log. So burn log level also needs info-level. The debug mode is not suitable for the server running mode.

Reply:
There are many users of this code and while I would like to help you I am not going to make their lives worse because of your particular setup.  You don't want to use debug logs because they are noisy but we would cause the exact same situation for other users by logging the burn info of every miner at the INFO level.

If you want to keep working on getting burn events logged you can either (1) merge debug logs and modify your setup to filter only what you need from them (2) come up with a design for logging at different levels based on actor address.  I think (2) will be more work but I can talk about it with you in an issue if you want to go that route.

Reply:
I think (2) feasible. thanks!   restore burnFunds log level to debug.